masschallenge/semantic-ui-theme
# AC-7597 Create mentor confirmation page

### Changes introduced in: [AC- 7597](https://masschallenge.atlassian.net/browse/AC-7597) :
- Added label and radio button overrides
### How to test
Follow the test instructions on the sibling PR on [accelerate](https://github.com/masschallenge/accelerate/pull/2615) and ensure that these changes are reflected
